.github/workflows/nodejs.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - name: npm install, build, and test
       run: |
-        npm ci
+        npm i
```

Comment:
`npm ci` was throwing this error:
```
npm ERR! bindings not accessible from watchpack-chokidar2:fsevents

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-09-19T17_48_05_189Z-debug.log
Error: Process completed with exit code 1.
```
It has to do with the fact that `package-lock.json` is being generated by a different host (my pc) and then the deps are being installed on a different host (github ci runner). More info [here](https://github.com/bazelbuild/rules_nodejs/issues/2452)